source/jobs/prerequisites/db2.txt

```diff
@@ -0,0 +1,73 @@
+.. _rm-prereq-db2:
+
+=========================================
+Configure Migration Prerequisites for DB2
+=========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To run sync jobs from a DB2 source database, the database requires
```

Comment:
Minor: I might suggest adding target or some other identifying information for the second database mentioned in the sentence.

```suggestion
To run sync jobs from a DB2 source database, the target database requires
```